app/Repositories/Interfaces/PayloadRepositoryInterface.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+namespace App\Repositories\Interfaces;
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.OpenBraceNotAlone</br>**Severity**: error</br>**File**: app/Repositories/Interfaces/PayloadRepositoryInterface.php</br>**Line**: 6</br>Opening interface brace must be on a line by itself